operator/cmd/operator/server.go

```diff
@@ -146,6 +161,11 @@ func run() {
 		log.Fatalf("Could not add all controllers to operator manager: %v", err)
 	}
 
+	// Record version of operator in metrics
+	metrics.Version.
+		With(metrics.OperatorVersionLabel.Value(version.Info.String())).
+		Record(1.0)
```

Comment:
not sure which you find more readable, but you could also just use `Increment()` here.

Reply:
Yes. `OperatorVersionLabel` is a gauge. So I thought using `Record` makes more sense that `Increment` which is usually for counters.